### PR TITLE
Don't apply _Py_OPCODE twice

### PR DESCRIPTION
**Pull request by AlekseiNikiforovIBM**
It's already applied in PyInstDecoder::opcode.
Applying it twice returns incorrect result on big endian systems.

This change fixes 14 tests in test/functorch/test_dims.py on big endian systems.

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

**Comment by AlekseiNikiforovIBM** [comment]
Commit is rebased.

**Comment by kit1980** [comment]
> Is anything blocking this pull request?

@AlekseiNikiforovIBM FYI after you get an approval review from a qualified reviewer, you can post "@pytorchbot merge" yourself to merge the PR.

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**
